pkg/stacker/convert.go

```diff
@@ -123,7 +129,7 @@ func (c *Converter) convertCommand(cmd *Command) error {
 
 		for _, line := range cmd.Value {
 			// patch some cmds
-			line = strings.ReplaceAll(line, "mkdir", "mkdir -p ")
+			line = strings.ReplaceAll(line, "mkdir", "mkdir -p")
```

Comment:
So if the dockerfile was doing

```
alias mkdir_p="mkdir -p $*"
mkdir_p /tmp/a
```
This will trip up.  Do we care?

Why is this needed - does a dockerfile actually do this?  If so, maybe doing a regexp replace for "\<mkdir\>" would at least be safer.

Reply:
Yes, we do need it to be compatible with docker behavior. But you also have a point.

Reply:
Simplest change maybe to 's/mkdir /mkdir -p /g'

Reply:
I was thinking something like

```
re := regexp.MustCompile('\\bmkdir\\b')
line  = re.ReplaceAllString(line, 'mkdir -p')

Reply:
Ok, your regexp suggestion/change went through

Reply:
cool, thanks.